action.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: mcaf-mcvs-golang-action
+description: |
+  The Mission Critical Acceleration Framework (MCAF)
+  Mission Critical Vulnerability Scanner (MCVS) Golang action.
+inputs:
+  golang-version:
+    description: |
+      The Golang version that has to be installed and used.
+    default: 1.21.5
+    required: true
+  golangci-lint-version:
+    description: |
+      The Golangci-lint version that has to be installed and used.
+    default: v1.55.2
+    required: true
+  golang-unit-tests-exclusions:
+    description: |
+      The Golang paths that should be excluded from unit testing.
+    default: ' '
+runs:
+  using: 'composite'
+  steps:
+    #
+    # install a certain golang version
+    #
+    - uses: actions/setup-go@v4.0.0
+      with:
+        go-version: ${{ inputs.golang-version }}
+        cache: false
+    #
+    # code security scanning
+    #
+    - uses: aquasecurity/trivy-action@0.14.0
+      with:
+        scan-type: 'fs'
+        scan-ref: '.'
+        exit-code: '1'
+        ignore-unfixed: true
+        severity: 'CRITICAL,HIGH'
+    #
+    # run golangci-lint
+    #
+    - uses: golangci/golangci-lint-action@v3.7.0
+      with:
+        args: |-
+          --config configs/.golangci.yml \
+          --out-format=colored-line-number \
+          --timeout 2m30s \
+          -v
+        version: ${{ inputs.golangci-lint-version }}
+    #
+    # run gosec
+    #
+    - uses: securego/gosec@v2.18.2
```

Comment:
Seems like this is integrated in golangci-lint:
https://golangci-lint.run/usage/linters/